### PR TITLE
Enable LES with EB and add Sigma model

### PR DESCRIPTION
**Pull request by esclapez**

### CODE REVIEW

**Comment by esclapez** [comment]
@baperry2 you can ignore the HIP CI failures, there are issues with getting the HIP dependencies in CI at the moment.

File: Docs/source/manual/Model.rst [hunk 1]
```diff
@@ -381,22 +381,39 @@ multigrid solvers. More details on how to use HYPRE is provided in control Secti
...
+modified turbulent transport coefficients. The basis of all these algebraic closures is to model the subgrid scale (sgs) viscosity with:
 
 .. math::
 
-   \mu_t = \overline{\rho} C_s^2 \bar{\Delta}^2 |\widetilde{S}|, \hspace{12pt}  \widetilde{S}_{ij} = \frac{1}{2} \left(\frac{\partial u_i}{\partial x_j} + \frac{\partial u_j}{\partial x_i} \right).
+    \nu_{t} = \nu_{sgs} = ( C_m \Delta )^2 \mathcal{D}(\boldsymbol{u})
+
+where :math:`C_m` is a model constant, :math:`\Delta` is the subgrid length scale (typically the grid size) and :math:`\mathcal{D}(\boldsymbol{u})`
+is a differential operator acting on the filtered velocity. :math:`\mu_t` is then calculated using the filtered density :math:`\mu_t = \overline{\rho} \cdot \nu_{t}`.
+In the `Smgorinsky <https://doi.org/10.1175/1520-0493(1963)091<0099:GCEWTP>2.3.CO;2>`_ model, :math:`C_m = C_s = 0.18` and :math:`\mathcal{D}` writes:
```
Comment (baperry2) [inline Docs/source/manual/Model.rst]: Fix spelling of Smagorinsky, also this link doesn't gets mangled when generating the HTML

File: Docs/source/manual/Model.rst [hunk 2]
```diff
@@ -381,22 +381,39 @@ multigrid solvers. More details on how to use HYPRE is provided in control Secti
...
+
+.. math::
+
+   \mathcal{D}_{s} = |\widetilde{S}| = sqrt(2 \widetilde{S}_{ij}\widetilde{S}_{ij}), \hspace{12pt} \widetilde{S}_{ij} = \frac{1}{2} \left(\frac{\partial u_i}{\partial x_j} + \frac{\partial u_j}{\partial x_i} \right).
 
 where filtered quantities are indicated with an overbar, Favre-filtered quantities are indicated with
-a tilde. In the WALE model, it is computed as:
+a tilde. In the `<WALE https://doi.org/10.1023/A:1009995426001>`_  model, :math:`C_m = C_w = 0.60` and :math:`\mathcal{D}` writes:
```
Comment (baperry2) [inline Docs/source/manual/Model.rst]: This link is also broken

**Review by baperry2 — APPROVED** [review]
Looks good!

**Pull request merged**